### ISSUE TITLE
display of votes left and end session missing above map

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#4982]
**URL:** https://meinberlin-dev.liqd.net/projekte/module/burgerinnenhaushalt-3-phasen-20/
**user:** user that has entered code
**expected behaviour:** on the map I can see how may votes I have left
**behaviour:** after entering the code I don't see the number of votes left when on the map.
**important screensize:**
**device & browser:** 
**Comment/Question:** when on the map and then entering the code we do lead the user to the list. But if user decides to navigate via map there is no indication on the votes left. Should we not show the "votes left" + "end session" also above the map?

Screenshot?
<img width="882" alt="Bildschirmfoto 2023-02-22 um 13 48 29" src="https://user-images.githubusercontent.com/35491681/220624021-e9b3feb8-9ee7-4ecf-b3a2-8d1277b8b485.png">

**Comment by fuzzylogic2000** [comment]
That was never in the designs or concept and the map is not in React. Also, you cannot see on the tiles what you have voted on and we will not add that. We shouldn't just also add that to the map without concept or testing.

**Comment by CarolingerSeilchenspringer** [comment]
@fuzzylogic2000 thanks, will take that back to client.
